src/ucp/wireup/address.c

```diff
@@ -410,9 +416,15 @@ ucp_address_gather_devices(ucp_worker_h worker, const ucp_ep_config_key_t *key,
             /* iface address length (+flags) can take 2 bytes with address
              * version 2 in non-unified mode
              */
-            dev->tl_addrs_size += ucp_address_packed_length_size(
+            length_size         = ucp_address_packed_length_size(
                                       worker, iface_attr->iface_addr_len,
                                       UCP_ADDRESS_IFACE_LEN_MASK, addr_version);
+            if (length_size < 0) {
+                ucs_free(devices);
```

Comment:
can we print the name of the tl/device that we failed to pack? so potentially user could disable it using UCX_TLS or UCX_NET_DEVICES

Reply:
added, but likely it's a list of them